components/homepage/components-section.tsx

```diff
@@ -25,6 +25,7 @@ export const ComponentsSection: FC = () => {
           ))}
         </div>
         <div className="mb-4 flex w-full justify-center text-center">
+          {/* @ts-expect-error TODO: fix `as` inference */}
```

Comment:
As you are fixing #962, is it necessary to have these @ts-expect-error comments?

Reply:
Yes, this is needed since the fix is reverting #1244.